src/main/java/uk/gov/hmcts/ecm/common/client/CcdClient.java

```diff
@@ -102,6 +103,31 @@ public CCDRequest startCaseCreationTransfer(String authToken,
         return restTemplate.exchange(uri, HttpMethod.GET, request, CCDRequest.class).getBody();
     }
 
+    public uk.gov.hmcts.et.common.model.ccd.CCDRequest startCaseMigrationToReform(String authToken, String jurisdiction,
+                                                                                  String caseTypeId) throws IOException {
+        HttpEntity<String> request = new HttpEntity<>(buildHeaders(authToken));
+        String uri = ccdClientConfig.buildStartCaseMigrationToReformUrl(userService.getUserDetails(authToken).getUid(),
+                jurisdiction, caseTypeId);
+        ResponseEntity<uk.gov.hmcts.et.common.model.ccd.CCDRequest> responseEntity =
+                restTemplate.exchange(uri, HttpMethod.GET, request, uk.gov.hmcts.et.common.model.ccd.CCDRequest.class);
+        return responseEntity.getBody();
+    }
+
+    public uk.gov.hmcts.et.common.model.ccd.SubmitEvent submitCaseCaseReform(String authToken,
```

Comment:
```suggestion
    public uk.gov.hmcts.et.common.model.ccd.SubmitEvent submitCaseCaseReform(String authToken,
```
public uk.gov.hmcts.et.common.model.ccd.SubmitEvent submitCaseForReformCaseType(String authToken,